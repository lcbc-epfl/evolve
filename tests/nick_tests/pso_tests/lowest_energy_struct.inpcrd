default_name
   304
  28.6650000  21.9200000   1.5210000  27.9220000  22.5810000   1.7000000
  28.3590000  21.3190000   0.7690000  29.5330000  22.3670000   1.2640000
  28.9110000  20.9840000   2.6310000  29.7640000  20.3710000   2.3400000
  29.3310000  21.6960000   3.9350000  30.2300000  22.2830000   3.7490000
  28.5400000  22.3590000   4.2870000  29.5570000  20.6420000   4.9950000
  29.9690000  19.4960000   4.8320000  29.3670000  20.9590000   6.2360000
  29.4180000  20.1950000   6.8940000  28.8940000  21.8150000   6.4880000
  27.6660000  20.0480000   2.8410000  26.7110000  20.5140000   3.5060000
  27.6950000  18.8110000   2.3010000  28.5200000  18.4860000   1.8170000
  26.4580000  17.9600000   2.4100000  26.1210000  18.1160000   3.4340000
  25.3100000  18.3760000   1.4600000  25.3810000  19.4440000   1.2540000
  25.3750000  17.7990000   0.5370000  23.8910000  17.9790000   1.8910000
  23.7720000  16.8950000   1.8870000  23.4670000  18.5840000   3.2330000
  22.4590000  18.3320000   3.5640000  24.0460000  18.0300000   3.9720000
  23.7430000  19.6370000   3.2800000  22.9170000  18.2920000   0.8190000
  22.8530000  19.3790000   0.7760000  23.3060000  17.8640000  -0.1050000
  22.0210000  17.7280000   1.0780000  26.8020000  16.4320000   2.2420000
  27.9030000  16.0490000   1.8530000  25.8380000  15.5180000   2.5280000
  24.9270000  15.8960000   2.7480000  26.0610000  14.0580000   2.5080000
  26.9450000  14.0130000   3.1440000  24.8880000  13.2870000   3.1640000
  25.2220000  12.2640000   3.3360000  24.7320000  13.5730000   4.2040000
  23.5400000  13.2500000   2.5050000  23.3320000  12.3450000   1.4290000
  24.1310000  11.7020000   1.0910000  22.0710000  12.3720000   0.7020000
  21.8650000  11.8050000  -0.1950000  21.0750000  13.2620000   1.1820000
  19.9330000  13.3780000   0.4680000  19.4620000  14.1410000   0.8100000
  21.2740000  14.1340000   2.2920000  20.6280000  14.9850000   2.4520000
  22.4750000  14.0580000   2.9760000  22.6250000  14.8040000   3.7430000
  26.4560000  13.3580000   1.2010000  27.2860000  12.4120000   1.1020000
  25.8960000  13.9060000   0.1540000  25.1930000  14.6230000   0.2640000
  26.2790000  13.5950000  -1.2070000  26.2380000  12.5070000  -1.2690000
  25.4130000  14.1240000  -2.4100000  25.7310000  13.7740000  -3.3920000
  24.0150000  13.4900000  -2.1960000  23.6680000  13.7210000  -1.1890000
  23.3530000  13.9220000  -2.9460000  24.1410000  12.4140000  -2.3190000
  25.3470000  15.6270000  -2.3950000  25.4030000  15.9260000  -1.3480000
  26.2190000  15.9730000  -2.9490000  24.2000000  16.3790000  -2.8790000
  24.3190000  17.4590000  -2.7940000  23.8710000  16.0740000  -3.8730000
  23.4600000  16.0480000  -2.1500000  27.7090000  13.9860000  -1.6170000
  28.1470000  13.4150000  -2.5930000  28.3640000  14.7910000  -0.7810000
  27.8370000  15.3280000  -0.1070000  29.7360000  15.2900000  -0.9000000
  30.1700000  15.1380000  -1.8880000  29.8670000  16.8090000  -0.5350000
  29.6120000  17.0110000   0.5060000  30.9080000  17.1180000  -0.6220000
  29.0620000  17.5490000  -1.4580000  29.4270000  17.3910000  -2.4730000
  28.0040000  17.3220000  -1.3300000  29.2220000  19.1060000  -1.1860000
  28.9290000  19.6280000  -0.1090000  29.7080000  19.9030000  -2.0920000
  29.8320000  20.8810000  -1.8760000  29.5660000  19.6900000  -3.0700000
  30.6170000  14.4470000   0.0570000  31.7050000  13.9770000  -0.3510000
  30.2170000  14.1670000   1.2860000  29.2920000  14.4590000   1.5670000
  30.9590000  13.3310000   2.2880000  31.9230000  13.8110000   2.4520000
  30.1090000  13.2280000   3.6630000  29.2230000  12.6780000   3.3450000
  30.6250000  12.5710000   4.3630000  29.7390000  14.4720000   4.3720000
  30.2010000  15.7180000   4.1330000  31.0380000  15.9420000   3.4870000
  29.4630000  16.7070000   4.8970000  29.6180000  17.7020000   4.9780000
  28.4240000  16.0960000   5.5230000  27.4250000  16.5590000   6.3920000
  27.4920000  17.6240000   6.5590000  26.4510000  15.7110000   6.9450000
  25.7170000  16.1020000   7.6340000  26.6130000  14.3490000   6.7160000
  25.9060000  13.6130000   7.0690000  27.7410000  13.8670000   6.0590000
  27.8590000  12.8180000   5.8340000  28.6600000  14.6810000   5.3650000
  31.2830000  11.9100000   1.8070000  32.3660000  11.4050000   2.1480000
  30.4650000  11.2570000   0.9780000  29.5260000  11.6290000   0.9910000
  30.6710000   9.8360000   0.5520000  31.0640000   9.2150000   1.3570000
  29.3120000   9.1360000   0.1140000  29.4730000   8.0950000  -0.1670000
  28.6300000   9.0730000   0.9620000  28.5810000   9.8230000  -1.0590000
  28.6800000  10.9090000  -1.0380000  28.9690000   9.1420000  -2.3780000
  28.3660000   8.2660000  -2.6170000  28.7740000   9.8490000  -3.1850000
  30.0460000   8.9850000  -2.4290000  27.0790000   9.5520000  -0.8870000
  26.5380000   9.8250000  -1.7940000  26.9100000   8.4890000  -0.7170000
  26.6520000  10.2030000  -0.1240000  31.8400000   9.7930000  -0.5140000
  32.3930000   8.7690000  -0.7670000  32.0560000  10.9040000  -1.2680000
  31.4480000  11.6960000  -1.1160000  33.0280000  11.0340000  -2.3340000
  33.2690000  10.0530000  -2.7430000  32.5520000  11.9690000  -3.4170000
  32.1860000  12.9280000  -3.0510000  33.3400000  12.1720000  -4.1420000
  31.2190000  11.4270000  -4.0580000  31.3630000  10.4210000  -4.4500000
  30.4990000  11.4150000  -3.2400000  30.7310000  12.4420000  -5.1630000
  30.2300000  13.2860000  -4.6880000  31.6280000  12.8740000  -5.6070000
  29.8190000  11.9670000  -6.3070000  29.7010000  12.7140000  -7.0920000
  30.4160000  11.1550000  -6.7210000  28.4000000  11.6640000  -5.9650000
  28.2520000  10.9700000  -5.2470000  28.0000000  12.5030000  -5.5700000
  27.8840000  11.4000000  -6.7930000  34.3610000  11.5630000  -1.8370000
  35.4280000  11.1710000  -2.3980000  34.3680000  12.4970000  -0.8900000
  33.4540000  12.8020000  -0.5860000  35.5100000  13.1650000  -0.3120000
  36.3150000  13.2390000  -1.0430000  35.2440000  14.5350000   0.1730000
  34.8170000  15.1400000  -0.6270000  34.4770000  14.5100000   0.9470000
  36.4360000  15.2240000   0.8440000  37.3760000  15.6460000   0.0990000
  36.4690000  15.2520000   2.0620000  36.2530000  12.3780000   0.8250000
  37.5050000  12.4120000   0.7910000  35.5710000  11.5970000   1.7420000
  34.5640000  11.6650000   1.7370000  36.2190000  10.6390000   2.7120000
  36.1510000   9.6170000   2.3400000  37.2810000  10.8660000   2.8100000
  35.6320000  10.6830000   4.0740000  36.2870000  11.0110000   5.0380000
  34.3940000  10.2730000   4.2280000  33.8540000  10.1030000   3.3920000
  33.6350000  10.3810000   5.4420000  32.8460000   9.6320000   5.3730000
  34.3520000  10.0890000   6.2090000  33.0180000  11.6990000   5.7440000
  33.4300000  12.6910000   5.1860000  32.1480000  11.7940000   6.7250000
  31.6280000  10.7210000   7.5820000  32.3970000  10.0310000   7.9270000
  30.8730000  10.1100000   7.0870000  30.9610000  11.3920000   8.7290000
  31.7260000  11.4870000   9.4990000  30.1090000  10.8750000   9.1720000
  30.6120000  12.7560000   8.1840000  30.3680000  13.5010000   8.9400000
  29.6830000  12.6920000   7.6170000  31.7420000  13.1250000   7.2520000
  31.3310000  13.6660000   6.3990000  32.8410000  13.9170000   7.9380000
  33.9430000  13.3670000   8.2600000  32.5690000  15.2270000   8.2540000
  31.6230000  15.5430000   8.0970000  33.5110000  16.2280000   8.7840000
  34.3800000  16.1220000   8.1350000  32.9660000  17.6100000   8.4300000
  32.1330000  17.7450000   9.1190000  33.7020000  18.3690000   8.6970000
  32.5090000  17.7840000   7.0230000  32.1180000  18.6500000   6.8890000
  33.8590000  16.1740000  10.2860000  34.1480000  17.2360000  10.8110000
  33.9450000  15.0490000  10.9220000  33.6780000  14.1670000  10.5090000
  34.3750000  14.9140000  12.3100000  33.7810000  15.6400000  12.8650000
  34.0170000  13.6010000  12.9510000  34.5780000  12.7230000  12.6320000
  34.1420000  13.7660000  14.0210000  32.5400000  13.3660000  12.8030000
  32.3070000  12.6160000  13.3540000  35.8650000  15.1010000  12.5830000
  36.6460000  14.2020000  12.3820000  36.2600000  16.3370000  12.9890000
  35.6000000  17.0680000  13.2110000  37.6080000  16.7340000  13.2490000
  37.6820000  17.5820000  13.9300000  38.0480000  15.8750000  13.7560000
  38.4470000  16.9340000  11.9730000  39.7250000  16.9550000  12.0650000
  37.7180000  17.0760000  10.8140000  36.7230000  17.2430000  10.8540000
  38.2740000  17.3200000   9.5120000  39.2700000  17.7490000   9.6270000
  38.4350000  15.9880000   8.8350000  39.0310000  16.1240000   7.9320000
  39.0100000  15.3340000   9.4900000  37.1980000  15.3510000   8.3020000
  36.4150000  15.1100000   9.0210000  36.7160000  16.0740000   7.6440000
  37.4240000  14.0470000   7.4840000  38.4040000  14.0230000   7.0080000
  37.3930000  13.2760000   8.2540000  36.3920000  14.0270000   6.4720000
  35.4710000  13.7070000   6.7360000  36.4400000  14.4750000   5.2230000
  37.4420000  15.1400000   4.6980000  38.2560000  15.4760000   5.1920000
  37.3450000  15.3440000   3.7130000  35.4220000  14.2580000   4.3950000
  34.5740000  13.8170000   4.7200000  35.4270000  14.6950000   3.4840000
  37.4560000  18.4140000   8.8270000  36.2330000  18.2840000   8.8570000
  38.0290000  19.4430000   8.2130000  39.4660000  19.6980000   8.0490000
  39.9450000  18.7770000   7.7160000  39.8490000  20.1110000   8.9820000
  39.5480000  20.8060000   7.0180000  39.4970000  20.2800000   6.0640000
  40.4830000  21.3630000   7.0750000  38.2490000  21.5760000   7.1940000
  37.9190000  21.9860000   6.2400000  38.2470000  22.3070000   8.0030000
  37.3000000  20.4720000   7.4940000  36.5630000  20.8660000   8.1940000
  36.4260000  20.0610000   6.2380000  36.5900000  18.9530000   5.6490000
  35.3490000  20.8050000   5.8950000  35.1510000  22.2130000   6.3480000
  35.9010000  22.9310000   6.0150000  35.1150000  22.2550000   7.4370000
  33.9030000  22.6470000   5.5880000  34.2450000  23.1210000   4.6680000
  33.2770000  23.3360000   6.1560000  33.1200000  21.3210000   5.4560000
  32.3380000  21.2890000   4.6970000  32.6670000  21.0630000   6.4130000
  34.2390000  20.3380000   5.0570000  33.8510000  19.3280000   5.1840000
  34.5770000  20.4340000   3.5250000  35.3420000  21.2610000   3.0730000
  33.8210000  19.6420000   2.6570000  32.7980000  18.6810000   3.0930000
  31.9660000  19.1970000   3.5730000  33.2420000  17.9540000   3.7730000
  32.2690000  18.0090000   1.7800000  31.2110000  18.2500000   1.6780000
  32.2560000  16.9200000   1.8280000  33.1470000  18.5380000   0.6860000
  32.5660000  18.7580000  -0.2100000  33.8750000  17.7490000   0.4930000
  33.7570000  19.8460000   1.2570000  34.6980000  19.9880000   0.7260000
  32.8200000  21.0080000   0.9150000  31.7890000  21.1720000   1.6290000
  33.2020000  21.8460000  -0.0270000  34.0220000  21.6330000  -0.5780000
  32.5780000  23.1400000  -0.2830000  32.7930000  23.8160000   0.5440000
  33.0570000  23.7720000  -1.5540000  32.6230000  23.3320000  -2.4510000
  32.6440000  24.7760000  -1.6570000  34.4620000  23.7950000  -1.6650000
  34.7300000  24.5410000  -1.1220000  31.0340000  23.1660000  -0.3740000
  30.3930000  23.8460000   0.4060000  30.4690000  22.3550000  -1.1450000
