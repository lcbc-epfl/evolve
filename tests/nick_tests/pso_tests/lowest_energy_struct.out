
          -------------------------------------------------------
          Amber 16 SANDER                              2016
          -------------------------------------------------------

| Run on 04/11/2018 at 13:22:34

|   Executable path: sander
| Working directory: /share/lcbcpc35/home/nbrownin/git/GABio/tests/nick_tests/pso_tests
|          Hostname: Unknown
  [-O]verwriting output

File Assignments:
|  MDIN: amber_minimization.in                                                 
| MDOUT: lowest_energy_struct.out                                              
|INPCRD: lowest_energy_struct.inpcrd                                           
|  PARM: lowest_energy_struct.prmtop                                           
|RESTRT: lowest_energy_struct.rst                                              
|  REFC: refc                                                                  
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: mdcrd                                                                 
|MDINFO: mdinfo                                                                
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

Minimize with implicit solvent, standard Born solvation model igb=1            
 &cntrl                                                                        
  imin = 1,                                                                    
  irest = 0,                                                                   
  ntmin = 1,                                                                   
  maxcyc = 1000,                                                               
  ncyc = 500,                                                                  
  ntpr = 200,                                                                  
  ntr = 0,                                                                     
  igb = 5,                                                                     
  ntb = 0,                                                                     
  cut = 999                                                                    
 /                                                                             

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 04/11/18 Time = 13:06:48
 NATOM  =     304 NTYPES =      12 NBONH =     150 MBONA  =     160
 NTHETH =     346 MTHETA =     219 NPHIH =     678 MPHIA  =     552
 NHPARM =       0 NPARM  =       0 NNB   =    1701 NRES   =      20
 NBONA  =     160 NTHETA =     219 NPHIA =     552 NUMBND =      32
 NUMANG =      70 NPTRA  =      37 NATYP =      21 NPHB   =       0
 IFBOX  =       0 NMXRS  =      24 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               29178
|     Hollerith            934
|     Integer            36591
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              374 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =     200, ntrx    =       1, ntwr    =       1
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       0, igb     =       5, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.80000, gbgamma =   4.85000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Energy minimization:
     maxcyc  =    1000, ncyc    =     500, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010
|  INFO: Old style inpcrd file read


--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -4.5701E+02     1.7984E+01     1.0062E+02     CG         83

 BOND    =       62.9275  ANGLE   =      153.7226  DIHED      =      219.7399
 VDWAALS =     -106.9139  EEL     =    -1518.2950  EGB        =     -230.0776
 1-4 VDW =       65.1161  1-4 EEL =      896.7703  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200      -6.7004E+02     7.9478E-01     7.1342E+00     CD2        53

 BOND    =       14.5038  ANGLE   =       42.4222  DIHED      =      192.2232
 VDWAALS =     -117.4320  EEL     =    -1523.6016  EGB        =     -231.6976
 1-4 VDW =       55.6698  1-4 EEL =      897.8732  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    400      -6.7545E+02     6.5600E-01     1.1524E+01     CG        165

 BOND    =       14.0573  ANGLE   =       40.8517  DIHED      =      190.5882
 VDWAALS =     -118.3474  EEL     =    -1525.0329  EGB        =     -231.4410
 1-4 VDW =       55.5343  1-4 EEL =      898.3443  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    600      -6.8380E+02     2.8020E-01     1.8811E+00     HD12       72

 BOND    =       13.2001  ANGLE   =       39.5416  DIHED      =      186.6444
 VDWAALS =     -119.2498  EEL     =    -1532.4432  EGB        =     -227.6042
 1-4 VDW =       56.1451  1-4 EEL =      899.9634  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    800      -6.8611E+02     1.8808E-01     1.2519E+00     C         302

 BOND    =       13.3642  ANGLE   =       39.7358  DIHED      =      186.0677
 VDWAALS =     -120.5934  EEL     =    -1538.1366  EGB        =     -223.8510
 1-4 VDW =       56.0433  1-4 EEL =      901.2607  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -6.8699E+02     1.8541E-01     1.6189E+00     C         302

 BOND    =       13.2973  ANGLE   =       39.9203  DIHED      =      185.9789
 VDWAALS =     -121.2476  EEL     =    -1540.5404  EGB        =     -222.0603
 1-4 VDW =       55.9183  1-4 EEL =      901.7425  RESTRAINT  =        0.0000


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -6.8699E+02     1.8541E-01     1.6189E+00     C         302

 BOND    =       13.2973  ANGLE   =       39.9203  DIHED      =      185.9789
 VDWAALS =     -121.2476  EEL     =    -1540.5404  EGB        =     -222.0603
 1-4 VDW =       55.9183  1-4 EEL =      901.7425  RESTRAINT  =        0.0000

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.00 ( 0.04% of Total)
|                Calc gb radii              1.21 (20.44% of Gen B)
|                Calc gb diag               2.47 (41.67% of Gen B)
|                Calc gb off-diag           2.24 (37.87% of Gen B)
|                Other                      0.00 ( 0.01% of Gen B)
|             Gen Born time              5.92 (100.0% of Nonbo)
|          Nonbond force              5.92 (92.75% of Force)
|          Bond/Angle/Dihedral        0.46 ( 7.21% of Force)
|          Other                      0.00 ( 0.03% of Force)
|       Force time                 6.38 (100.0% of Runmd)
|    Runmd Time                 6.38 (74.81% of Total)
|    Other                      2.14 (25.15% of Total)
| Total time                 8.53 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 13:22:34.606  on 04/11/2018
|           Setup done at 13:22:34.625  on 04/11/2018
|           Run   done at 13:22:43.132  on 04/11/2018
|     wallclock() was called   18028 times
